### PR TITLE
add bootstrap and bootstrap's navbar into project

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #111

### CODE REVIEW

File: finance-tracker/app/views/layouts/_navigation.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
```
Comment (tlkiong) [inline finance-tracker/app/views/layouts/_navigation.html.erb:1]: What does this have to do with adding bootstrap as per your PR title suggest?

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: finance-tracker/app/views/layouts/_navigation.html.erb (hunk 1 above)
Comment (LeeRenJie) [reply finance-tracker/app/views/layouts/_navigation.html.erb:1]: this is bootstrap's navbar code

**Comment by tlkiong** [comment]
I know what it is. Your pr title say is add bootstrap. Never said about
adding navbar

**Review by tlkiong — APPROVED** [review]
